ccd-definition/ComplexTypes/CareSupervision/2_Others.json

```diff
@@ -47,18 +47,28 @@
     "LiveFrom": "01/01/2017",
     "ID": "Others",
     "ListElementCode": "addressKnow",
-    "FieldType": "YesOrNo",
+    "FieldType": "FixedRadioList",
+    "FieldTypeParameter": "IsAddressKnowList",
```

Comment:
I guess just a note for testing, maybe worth setting up an old case with a Yes/No answer on the case already, to confirm this works seamlessly!